frontend/class-opengraph.php

```diff
@@ -826,6 +829,22 @@ private function get_front_page_image() {
 	}
 
 	/**
+	 * Get the images of the posts page.
+	 */
+	private function get_posts_page_image() {
```

Comment:
The name confused me a bit. I personally expect functions starting with the word "get" to return something. In this case, I expected this function to get the image to use for the static posts page, instead it also sets this. Admittedly the same holds true for the get_front_page_image function I see above this one. Do you think a naming change would be appropriate?